launchers/common/base-mds/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ dependencies {
 
     // Currently there are no mds-specific EDC extensions
```

Comment:
Why is there no mds-specific EDC extension? In line 11 an extension is imported

Reply:
Hi @SebastianOpriel, I believe this issue has been resolved, but unfortunately, I do not have the permissions to mark it as such.